apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/FissClaimTransformer.java

```diff
@@ -31,21 +32,27 @@
           FissClaim::getCurrStatusEnum,
           FissClaim::hasCurrStatusUnrecognized,
           FissClaim::getCurrStatusUnrecognized,
-          FissClaimStatus.UNRECOGNIZED);
+          FissClaimStatus.UNRECOGNIZED,
+          ImmutableSet.of(),
+          ImmutableSet.of(EnumStringExtractor.Options.RejectUnrecognized));
   private static final EnumStringExtractor<FissClaim, FissProcessingType> currLoc1 =
       new EnumStringExtractor<>(
           FissClaim::hasCurrLoc1Enum,
           FissClaim::getCurrLoc1Enum,
           FissClaim::hasCurrLoc1Unrecognized,
           FissClaim::getCurrLoc1Unrecognized,
-          FissProcessingType.UNRECOGNIZED);
+          FissProcessingType.UNRECOGNIZED,
```

Comment:
I missed this from before, but are BOTH currLoc1 and currLoc2 enums?

Reply:
Yes they are.  Do we want to validate them too then?  It would be easy to do now...

Reply:
Not yet, I want to wait till we have a use case for them in our FHIR responses.  For now we should accept any value